### DISCUSSION TITLE
Dead links and outdated information

### DISCUSSION THREAD

**Discussion by joachimmetz** [forensicswiki/wiki#54] [Q&A]
What do we do with dead links and out dated information?

I propose:

* to remove references to dead links if no alternative is available - https://github.com/forensicswiki/wiki/pull/55
* to remove outdated information - https://github.com/forensicswiki/wiki/pull/52

Alternatives:

* archive outdated information for historical purposes
* remove dead link but keep reference for historical purposes

**Comment by RyanDFIR** [comment]
I also favor removing the dead links & outdated information from the live site. Since we've moved to a git-based system, all the old links / info / etc are still preserved; it would just be harder to find (need to go back through the old commits on GitHub). I think having a live archive section on the site could confuse people. 

I think deleting the outdated stuff and making a one-pager on how to search through old commits for a specific topic to see removed / obsolete info would be a good  way to go.
